src/main/java/org/truffleruby/debug/TruffleDebugNodes.java

```diff
@@ -254,7 +254,7 @@ private void printAst(CallTarget callTarget) {
             if (callTarget instanceof RootCallTarget) {
                 printAst((RootCallTarget) callTarget);
             } else {
-                throw new RaiseException(getContext().getCoreExceptions().internalError("call target is not a root call target", this));
+                throw new RaiseException(getContext().getCoreExceptions().internalError("call target is not a root call target", this), true);
```

Comment:
We should just do the cast directly, I don't there is any case where we have a non-RootCalLTarget.

Reply:
Fixing.